auth-web/src/components/pay/eft/ShortNameTransactions.vue

```diff
@@ -0,0 +1,168 @@
+  props: {
+    shortName: {
+      type: Object,
+      default: () => ({
+        shortName: null,
+        transactionDate: null
+      })
+    }
+  },
+  setup (props) {
+    const headers = [
+      {
+        col: 'transactionDate',
+        hasFilter: false,
+        minWidth: '125px',
+        value: 'Payment Received Date'
+      },
+      {
+        col: 'depositAmount',
+        hasFilter: false,
+        minWidth: '125px',
+        value: 'Amount'
+      }
+    ]
+
+    const state = reactive({
+      results: [],
+      totalResults: 1,
+      filters: {
+        isActive: false,
+        pageNumber: 1,
+        pageLimit: 5
+      } as EFTTransactionFilterParams,
+      loading: false,
+      shortName: {
+        id: null
+      }
+    })
+
+    function setShortName (shortname) {
+      state.shortName = shortname
+      loadTransactions(state.shortName.id)
+    }
+
+    watch(() => props.shortName, () => setShortName(props.shortName), { deep: true })
+    async function loadTransactions (shortnameId: string): Promise<void> {
+      try {
+        state.loading = true
+        const response = await PaymentService.getEFTTransactions(shortnameId, state.filters)
+        if (response?.data) {
+          /* We use appendToResults for infinite scroll, so we keep the existing results. */
+          state.results = response.data.items
+          state.totalResults = response.data.total
+        } else {
+          throw new Error('No response from getEFTTransactions')
+        }
+      } catch (error) {
+        // eslint-disable-next-line no-console
+        console.error('Failed to getEFTTransactions list.', error)
+      }
+      state.loading = false
+    }
+
+    function updateFilter () : void { }
+
+    const tableDataOptions: Ref<DataOptions> = ref(_.cloneDeep(DEFAULT_DATA_OPTIONS) as DataOptions)
+
+    watch(() => state.filters.pageNumber, (val: number) => { tableDataOptions.value.page = val })
+    watch(() => tableDataOptions.value, (val: DataOptions) => {
+      const newPage = val?.page || DEFAULT_DATA_OPTIONS.page
+      const newLimit = val?.itemsPerPage || DEFAULT_DATA_OPTIONS.itemsPerPage
+      // need this check or vi test continuously loops on initialization
+      if (state.filters.pageNumber !== newPage || state.filters.pageLimit !== newLimit) {
+        state.filters.pageNumber = val?.page || DEFAULT_DATA_OPTIONS.page
+        state.filters.pageLimit = val?.itemsPerPage || DEFAULT_DATA_OPTIONS.itemsPerPage
+        loadTransactions(state.shortName?.id)
+      }
+    })
+
+    const formatDate = CommonUtils.formatDisplayDate
+    const formatCurrency = CommonUtils.formatAmount
+
+    return {
+      formatCurrency,
+      formatDate,
+      headers,
+      tableDataOptions,
+      state,
+      updateFilter
+    }
+  }
+})
+</script>
+
+<style lang="scss" scoped>
+@import '@/assets/scss/theme.scss';
+
+.card-title {
+  background-color: $app-lt-blue;
+  justify-content: left;
+}
+
+::v-deep {
```

Comment:
are there special stylings that aren't available for other tables? if so this makes sense, if not we should use a SCSS to build a generalized styling

Reply:
removed most of this now